core/dbt/include/global_project/macros/etc/column_helpers.sql

```diff
@@ -0,0 +1,59 @@
+{% macro column_list(columns) %}
+  {%- for col in columns %}
+    {{ col.name }} {% if not loop.last %},{% endif %}
+  {% endfor -%}
+{% endmacro %}
```

Comment:
perhaps justifying this PR further, but this is the first time I've seen this macro before... ah! `/global_project/macros/materializations/helpers.sql`! guess i never looked there before... now I'm racking my brain for instances of this macro not being used where it should be.

Reply:
Honestly, we've probably reinvented this wheel a dozen+ times. I didn't realize it existed until I did the work for this PR either. We have some utilities in core that we've used for various materializations, but some of them aren't even being used anymore. At this point, we could delete them entirely, or to keep them around forever—either feels like a valid approach.

Reply:
I don't see `column_list` or `column_list_for_create_table` being used in any of our global or plugin macros/materializations, so I think I'm just going to snip these out